### ISSUE TITLE
Repeater on Shapes creating artefacts (again)

### ISSUE THREAD

**Issue by kRVRKV** [airbnb/lottie-web#2543]
**Tell us about your environment**
Windows 10; latest bodymovin and lottiefiles plugins (downloaded through creative cloud)

**Browser and Browser Version:**
Google Chrome (90.0.4430.93 64bit)
MS Edge (90.0.818.62)

**After Effects Version:**
2021 v18.2

**What did you do? Please explain the steps you took before you encountered the problem.**
I've been creating the loading circle animation inside of the AE. I used Repeater on the Shape Layer to achieve the effect I was looking for. If I'd gone with the manual duplication - it wouldn't be as accurate. When I was finished I decided to export it as a "Demo" (because I need an SVG) using bodymovin.

**What did you expect to happen?**
I've seen that Repeater has been an issue in the past. But it seems that my case is something totally new.
I expected the export to be as identical as possible to what I had in AE.

**What actually happened? Please include as much _relevant_ detail as possible.**
The exported file got previewed on Chrome and Edge and showed a major difference in the animation. In other words, the circle's stroke either got an empty gap in the middle or got separated into two individual pieces. This also affects how the animation plays out.
After a lot of experimentation, I can surely say that it's the Repeater effect that creates it. Without the Repeater, the export is perfectly fine. If you try to add the Fill effect to the Shape Layer (while the Repeater is also enabled) on export you will notice that the artefact is actually a separate stroke that trails behind the main stroke.
This happens through the LottieFiles' preview as well.

**Please provide a download link to the After Effects file that demonstrates the problem.**
I've also included both demos (svgs) with and without the Repeater effect.
[issue-files.zip](https://github.com/airbnb/lottie-web/files/6477199/issue-files.zip)

**Comment by bodymovin** [comment]
hi, I have to look into it, but in the meantime, it seems that if you put the trim after the repeater, things work correctly

**Comment by kRVRKV** [comment]
> hi, I have to look into it, but in the meantime, it seems that if you put the trim after the repeater, things work correctly

Do you do any additional manipulations? Because if I put trim after the repeater (or repeater before trim) I get this result:
![image](https://user-images.githubusercontent.com/26649447/118445075-a3b9b980-b6f6-11eb-86bc-005d704fcbaa.png)

**Comment by bodymovin** [comment]
the only thing I did was move the repeater before the trim and it worked. Did you try it on the same "test.aep" project you shared before?

**Comment by kRVRKV** [comment]
> the only thing I did was move the repeater before the trim and it worked. Did you try it on the same "test.aep" project you shared before?

Yep, and inside of my main project as well. I tried many different manipulations using this method of moving effects and nothing worked so far. Not sure why. But I duplicated and aligned these loaders manually because it was a bit urgent for the project. If you feel it's worth researching further I'd be happy to help with any tests, files or logs :)
